src/op_mode/ssh.py

```diff
@@ -0,0 +1,104 @@
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import sys
+import glob
+import vyos.opmode
+from vyos.utils.process import cmd
+from vyos.configquery import ConfigTreeQuery
+
+def show_fingerprints(raw: bool, ascii: bool):
+    config = ConfigTreeQuery()
+    if not config.exists("service ssh"):
+        if raw:
+            return {"response": "SSH server is not enabled."}
+        else:
+            return "SSH server is not enabled."
+    else:
+        # Get list of server public keys
+        publickeys = glob.glob("/etc/ssh/*.pub")
+
+        if publickeys:
+            keys = []
+            for keyfile in publickeys:
+                if ascii:
+                    try:
+                        keys.append(cmd("ssh-keygen -l -v -E sha256 -f " + keyfile))
+                    except:
+                        # Ignore invalid public keys
+                        pass
+                else:
+                    try:
+                        keys.append(cmd("ssh-keygen -l -E sha256 -f " + keyfile))
+                    except:
+                        # Ignore invalid public keys
+                        pass
+            if raw:
+                return {"response": keys}
+            else:
+                response = "SSH server public key fingerprints:\n"
+                for key in keys:
+                    response = response + key + "\n"
+                return response
+        else:
+            if raw:
+                return {"response": "No SSH server public keys are found."}
+            else:
+                return "No SSH server public keys are found."
+
+def show_dynamic_protection(raw: bool):
+    config = ConfigTreeQuery()
+    if not config.exists("service ssh dynamic-protection"):
+        if raw:
+            return {"response": "SSH server dynamic-protection is not enabled."}
+        else:
+            return "SSH server dynamic-protection is not enabled."
+    else:
+        attackers = []
+        try:
+            # IPv4
+            attackers = attackers + json.loads(cmd("sudo nft -j list set ip sshguard attackers"))["nftables"][1]["set"]["elem"]
+        except:
+            pass
+        try:
+            # IPv6
+            attackers = attackers + json.loads(cmd("sudo nft -j list set ip6 sshguard attackers"))["nftables"][1]["set"]["elem"]
+        except:
+            pass
+        if attackers:
+            if raw:
+                return {"response": attackers}
```

Comment:
As I understand we don’t need extra “response” here 
@jestabro what do you think?

Reply:
I have removed the "response" from the raw sections.